test/e2e/storage/volume_metrics.go

```diff
@@ -123,3 +175,22 @@ func getControllerStorageMetrics(ms metrics.ControllerManagerMetrics) map[string
 	}
 	return result
 }
+
+// Verifies the specified metrics are in `kubeletMetrics`
```

Comment:
You're verifying that the metrics exist and the pvc-name and ns match what's expected. Are there any other metrics to test?

Reply:
Yes - The test is specifically verifying the new feature we added to tag metrics with pvc-name and ns. 
Verifying the metrics itself is covered by e2e tests under `e2e_node/summary_tests.go`.

Given that we now have this test - it would make sense to move/duplicate those tests with a PVC here. I can do that in a follow up PR.